.travis.yml

```diff
@@ -22,18 +17,29 @@ env:
   - MATRIX_TOXENV=integration-dynamodb
 matrix:
   include:
+  - python: '2.7'
+  - python: '3.4'
+  - python: '3.5'
   - python: '3.6'
+  #- python: '3.7'
+  #  dist: xenial    # required for Python 3.7 (travis-ci/travis-ci#9069)
```

Comment:
Should this be uncommented? This is the test for Python 3.7 environment, right?

Reply:
It is commented out because it is test 4 of the failing tests.  Python 3.7 unit fails.